libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/restarea/model/RestAreaGuideMapError.kt

```diff
@@ -0,0 +1,11 @@
+package com.mapbox.navigation.ui.maps.guidance.restarea.model
+
+/**
+ * The state is returned if there is an error retrieving service/parking area guide map.
+ * @param message String
+ * @param throwable an optional throwable value expressing the error
+ */
+class RestAreaGuideMapError internal constructor(
```

Comment:
Should we add `toString`, `hashCode` and `equals`? Not sure about the last two but I think that `toString` would be useful.